CustomerIOMessagingPushFCM.podspec

```diff
@@ -21,4 +21,9 @@ Pod::Spec.new do |spec|
   spec.module_name = "CioMessagingPushFCM" # the `import X` name when using SDK in Swift files 
   
   spec.dependency "CustomerIOMessagingPush", "= #{spec.version.to_s}"  
+
+  # Add FCM SDK as a dependency, as our SDK is designed to be compatible with it. 
+  # No version is specified which means that by default, the latest version is installed for customers. 
+  # Customers can override this by adding the pod to their Podfile themselves to specify a version they want to use. 
+  spec.dependency "FirebaseMessaging"
```

Comment:
Does this mean that the installer would give priority to the version mentioned in the app's podfile over latest version of FirebaseMessaging. 

Also, bit curious to learn, how would this make our(devs/QA) lives easier? One point i can think of is that the this will help us test the compatibility of latest FirebaseMessaging dependency with our SDK, is there any other point to support this change ?